src/features/user/ManageProjects/components/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { styled } from '@mui/material';
```

Comment:
This file seems to be introduced inadvertently. Should it belong to a different branch?

Reply:
yes, it's belongs to different branch, I'll make it correct